src/Microsoft.TemplateEngine.Cli/PostActionProcessors/ProcessStartPostActionProcessor.cs

```diff
@@ -32,6 +32,14 @@ public bool Process(IEngineEnvironmentSettings environment, IPostAction actionCo
             {
                 redirectStandardOutput = false;
             }
+            bool redirectStandardError = true;
+            // By default, standard error is redirected.
+            // Only redirect when the configuration says "redirectStandardError = false"
+            if (actionConfig.Args.TryGetValue("redirectStandardError", out string? redirectStandardErrorString)
+                    && string.Equals(redirectStandardErrorString, "false", StringComparison.OrdinalIgnoreCase))
```

Comment:
minor: consider `bool.TryParse`
https://docs.microsoft.com/en-us/dotnet/api/system.boolean.tryparse?view=net-5.0#System_Boolean_TryParse_System_String_System_Boolean__

Reply:
this is a legacy code and I think it's done so due to we want to have "false" written in configuration, not missing or empty value which may default to false. I would prefer to keep it this way.